src/middleware/api/auth.js

```diff
@@ -9,7 +9,7 @@ const getUserById = async (id) => {
 }
 const apiAuth = async (request, response, next) => {
     try{
-        const token = request.headers.authorization.split(' ')[1];
+        const token = request.headers.authorization;
```

Comment:
Is there a reason you changed this back?

Reply:
The bearer was not working initially and I thought we changed it back when we were reviewing the code.